boards/z1/Makefile.include

```diff
@@ -1,8 +1,6 @@
 # set default port depending on operating system
 PORT_LINUX ?= /dev/ttyUSB0
 PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
-# setup serial terminal
-include $(RIOTMAKE)/tools/serial.inc.mk
```

Comment:
Same here the serial port is used for programming as well.

Reply:
For this one, I agree it should be kept